kubernetes/config.go

```diff
@@ -5,13 +5,12 @@ import (
 	"path/filepath"
 
 	"github.com/docker/docker/pkg/homedir"
-	restclient "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
 // NewKubernetesConfig resolves the path to the desired Kubernetes configuration file, depending
 // environment variable and command line flag.
-func NewKubernetesConfig(configFlag string) (*restclient.Config, error) {
+func NewKubernetesConfig(configFlag string) clientcmd.ClientConfig {
```

Comment:
not new, but the `configFlag` variable is a bit confusing (i.e., it's not the flag itself, but a custom path, which _happens_ to be passed through a flag). Perhaps renamed this to (e.g.) `configPath`

Reply:
Yes, changing this.